README.md

```diff
@@ -31,3 +31,11 @@ As much as I *REALLY* hate using notebooks (and I think they should never be use
 ## The Data
 I have cleaned the data from the google sheet (https://docs.google.com/spreadsheets/d/1clTSVXBHpFB_4yqkAQhzvsDoVax4SZUYpGdbCy7FZ6U/edit#gid=329859720) a little and it should be easily readable using a Pandas dataframe.
 See the Python notebooks in `lib` to do this.
+
+## Future Science Saturdays
```

Comment:
Maybe we should make issues and vote (via thumbs) on the next Saturday. Thoughts?